src/python/test/test_dtm.py

```diff
@@ -91,11 +91,12 @@ def test_density():
 
 def test_dtm_overflow_warnings():
     pts = numpy.array([[10., 100000000000000000000000000000.], [1000., 100000000000000000000000000.]])
-
+    impl_warn = ["keops", "hnsw"]
     with warnings.catch_warnings(record=True) as w:
-        # TODO Test "keops" implementation as well when next version of pykeops (current is 1.5) is released (should fix the problem (cf. issue #543))
-        dtm = DistanceToMeasure(2, implementation="hnsw")
-        r = dtm.fit_transform(pts)
-        assert len(w) == 1
-        assert issubclass(w[0].category, RuntimeWarning)
-        assert "Overflow" in str(w[0].message)
+        for impl in impl_warn:
+            dtm = DistanceToMeasure(2, implementation=impl)
+            r = dtm.fit_transform(pts)
+        assert len(w) == 2
+        for i in range(len(w)):
+            assert issubclass(w[i].category, RuntimeWarning)
+            assert "Overflow" in str(w[i].message)
```

Comment:
Wouldn't it be a bit more precise to pull the loop outside, and test for each impl that we get exactly one warning?